Utilities/Pipelines/libHttpClient.CI.yml

```diff
@@ -0,0 +1,168 @@
+# example, "libHttpClient.CI_20210214.1".
+# Refs:
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/run-number
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables
+name: $(Build.DefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+
+jobs:
+
+  ####################
+  # Visual Studio 2017
+  ####################
+
+  - job: Win32Vs17Build
+    displayName: libHttpClient Win32/UWP VS2017 Build
+    pool:
+      name: xbl-1es-vs2017
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        x86_Debug:
+          Platform: x86
+          Configuration: Debug
+        x64_Debug:
+          Platform: x64
+          Configuration: Debug
+        x86_Release:
+          Platform: x86
+          Configuration: Release
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+    steps:
+      - template: Tasks/vs2017-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+
+  ####################
+  # Visual Studio 2019
+  ####################
+
+  - job: Win32Vs19Build
+    displayName: libHttpClient Win32/UWP VS2019 Build
+    pool:
+      name: xbl-1es-vs2019
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        x86_Debug:
+          Platform: x86
+          Configuration: Debug
+        x64_Debug:
+          Platform: x64
+          Configuration: Debug
+        x86_Release:
+          Platform: x86
+          Configuration: Release
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+    steps:
+      - template: Tasks/vs2019-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+
+  ####################
+  # Android
+  ####################
+
+  - job: AndroidBuild
+    displayName: libHttpClient Android Build
+    pool:
+      vmImage: internal-macos-10.15
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+        x64_Debug:
+          Platform: x64
+          Configuration: Debug
+    steps:
+      - template: Tasks/android-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+    
+  ####################
+  # iOS
+  ####################
+
+  - job: iOSBuild
+    displayName: libHttpClient iOS Build
+    pool:
+      vmImage: internal-macos-10.15
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        Debug:
+          Configuration: Debug
+    steps:
+      - template: Tasks/ios-build.yml
+        parameters:
+          configuration: $(Configuration)
+    
+  ####################
+  # XDK
+  ####################
+
+  - job: XDKBuild
+    displayName: libHttpClient XDK Build    
+    pool:
+      name: xbl-1es-vs2017-xdk
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        Durango_Release:
+          Platform: Durango
```

Comment:
I don't think we need to specify `Platform` here since I think the `xdk-build.yml` hard-codes Durango